src/client/opamLockCommand.ml

```diff
@@ -96,7 +96,8 @@ let select_packages atom_locs st =
                atoms
            in
            if missing <> [] then
-             (OpamConsole.error
+             (OpamConsole.error_and_exit
+                `Aborted
                 "Skipping %s, dependencies are not satisfied in this switch, \
                  not installed packages are:\n%s"
```

Comment:
The error message doesn't correspond to the action anymore (skipping vs. aborting)

Reply:
done

Reply:
my new error msg might not make any sense grammatically,

 as English is not  first language help a bit, pls.